pvt/benchmarks/joinExit.ts

```diff
@@ -48,7 +48,7 @@ async function main() {
   }
   console.log('\n');
 
-  for (let numTokens = 40; numTokens <= 90; numTokens += 10) {
+  for (let numTokens = 40; numTokens <= 80; numTokens += 10) {
```

Comment:
Consider defining this as a constant at the top, since you're going to have to keep changing it.

Reply:
I have 3: minRange, maxRange, and overall max